spec/server.test.js

```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const app = require('../server/app');
+
+describe('Test the root path', () => {
```

Comment:
This doesn't seem like an accurate title - only two of the tests are testing the root path. I would include the word server or api in this test as well.